src/components/Editor/index.jsx

```diff
@@ -96,6 +97,11 @@ const Editor = (
     isDisabled: isAttachmentsUploading,
   };
 
+  const debouncedOnChangeHandler = useFuncDebounce(
+    ({ editor }) => onChange(editor.getHTML()),
+    500
+  );
```

Comment:
The issue with adding a throttle is someone submitting the form before the throttled `onChange` is complete. Let's keep the number as low as `100` so that the `onChange` method is not triggered for each keystroke.

```suggestion
  const debouncedOnChangeHandler = useFuncDebounce(
    ({ editor }) => onChange(editor.getHTML()),
    100
  );
```